### PR TITLE
Api sidebar

### PR DESCRIPTION
**Pull request by adilansari**

### CODE REVIEW

File: protodocs/server/v1/observability.proto.mdx [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+
+_**path** server/v1/observability.proto_
+
+_**package** tigrisdata.observability.v1_
+
+---
+
+## Services
+
+### `HealthAPI`
```
Comment (efirs) [inline protodocs/server/v1/observability.proto.mdx:18]: This looks misplaced.

File: protodocs/server/v1/admin.proto.mdx [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+        oneofdecl: "",
+        defaultValue: "",
+      },
+    ],
+  }}
+/>
+
+---
+
+## Services
```
Comment (efirs) [inline protodocs/server/v1/admin.proto.mdx:194]: @himank Do we want to move services up? 
It requires manual work each time we regenerate API docs.

File: protodocs/server/v1/observability.proto.mdx (hunk 1 above)
Comment (adilansari) [reply protodocs/server/v1/observability.proto.mdx]: fixed

File: protodocs/server/v1/admin.proto.mdx (hunk 2 above)
Comment (himank) [reply protodocs/server/v1/admin.proto.mdx:194]: We can skip it for now as we are doing a lot of iterations on docs. Services at the top look nice and it is usually before other things. But it is fine to ignore it for now and we can fix it once we wrap up the documentation.

Comment (efirs) [reply protodocs/server/v1/admin.proto.mdx:194]: 👍🏻

**Review by efirs — APPROVED** [review]

**Pull request merged**